packages/web-components/src/components/va-button/va-button.scss

```diff
@@ -1,10 +1,25 @@
+@forward 'settings';
+
+@use 'usa-button/src/styles/usa-button';
+
+@import '../../global/formation_overrides';
+
+:host([disabled]:not([disabled='false'])) button {
+  pointer-events: none;
+}
+
+:host([disabled]:not([disabled='false'])) {
+  cursor: not-allowed;
+}
```

Comment:
The [USWDS version](https://designsystem.digital.gov/components/button/) did not have any cursor or pointer events disabled for the `disabled` button which I thought was a little strange so I am adding it here to work both for v1 and v3.